app/views/templates/app_layout.jst

```diff
@@ -0,0 +1,3 @@
+<div role="toolbar"></div>
+<div role="content"></div>
+<div role="player"></div>
```

Comment:
Should be `id="player"` instead of a role (you can probably safely use an `is` here as long as there's only one player insatnciated at a time)